app/controllers/api/v0/service_plans_controller.rb

```diff
@@ -4,6 +4,13 @@ class ServicePlansController < ApplicationController
       include Api::Mixins::IndexMixin
       include Api::Mixins::ShowMixin
 
+      def order
+        service_plan = model.find(params[:service_plan_id])
+        catalog_parameters = params[:service_parameters].permit!.merge(params[:provider_control_parameters].permit!)
+        task_id = service_plan.order(params[:catalog_id], catalog_parameters)
```

Comment:
> `params[:catalog_id]`

@mkanoor we have a `ServicePlan` belong_to a `ServiceOffering` so I think you passing in the catalog_id isn't needed

Reply:
@agrare Yes that is old stuff and not needed anymore